src/app/core/ui/navigation/menu-item/menu-item.component.ts

```diff
@@ -21,6 +22,7 @@ import { MatMenuModule } from "@angular/material/menu";
     Angulartics2Module,
     NgForOf,
     MatMenuModule,
+    DragDropModule,  // Added DragDropModule for drag-and-drop support
```

Comment:
same here: probably drag&drop is only needed in the admin components now in the current implementation?